### PR TITLE
Fix #60

### LINKED ISSUE #60: Change the image naming format to Yunohost image format
Current format:
`labriqueinternet_A20LIME2_2019-02-19_stretch.img`

Yunohost format:
`yunohost-stretch-3.4.2-lime-stable.img`

We could go for:
`internetcube-stretch-3.4.2-lime2-stable.img`

### PR DESCRIPTION
**Pull request by keomabrun**

### CODE REVIEW

**Review by alexAubin — APPROVED** [review]
LGTM

To go one step further (mid-term stuff) : we were wondering when first creating images from yunohost's armbian images if we shouldn't rework this script and merge it with the other one that transforms yunohost images to internetcube images.

So this script would "simply" download images from yunohost and transform them on the fly into internetcube images, injecting the hypercube script inside it. I think that might help making developing the whole hypercube thing easier as you don't need anymore to have two separate stuff (building the image with one script, flashing the SD card)

File: install-sd.sh [hunk 1]
```diff
@@ -255,13 +255,13 @@ function check_args() {
       exit_usage "Filename given to -f does not contain _encryptedfs_ in its name, but -e was set"
     fi
 
-    if [[ "${opt_imgpath}" =~ LIME2 ]]; then
+    if [[ "${opt_imgpath}" =~ lime2 ]]; then
```
Comment (Psycojoker) [inline install-sd.sh:258]: I don't know if it's already the case but maybe do a case insensitive match?

**Comment by keomabrun** [comment]
Good point Aleks. However the post-install still takes 40min on a lime2 and I am still hoping we could customize the images a bit more (pre-post-install). And I believe this cannot be done in the `install-sd` script.

**Comment by alexAubin** [comment]
Yes indeed :wink: That's kinda why we ended up not reworking the script so far (waiting to see how the pre-post-install ideas goes)

**Pull request merged**